src/test/coins_tests.cpp

```diff
@@ -558,16 +560,29 @@ BOOST_AUTO_TEST_CASE(ccoins_serialization)
     }
 }
 
+const static char DIRTY = CCoinsCacheEntry::DIRTY;
+const static char FRESH = CCoinsCacheEntry::FRESH;
+const static char NO_ENTRY = -1;
+
+struct CoinEntry {
+    const CAmount value;
+    const char flags;
+
+    CoinEntry(CAmount v, char s) : value(v), flags(s) {}
+
+    bool operator==(const CoinEntry& o) const = default;
+    friend std::ostream& operator<<(std::ostream& os, const CoinEntry& e) { return os << e.value << e.flags; }
```

Comment:
In commit "test: Migrate GetCoinsMapEntry to return MaybeCoin" (a902af6fb195fae97a89876b88d5034162ab8366)

Should this include a space between that value and flags? Otherwise if test fails it looks like amount and flags will run together and might not be possible to distinguish.

Reply:
Done